src/__tests__/FamilyIntake.test.js

```diff
@@ -0,0 +1,37 @@
+import FamilyIntake from '../components/common/FamilyIntake';
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+// mocking matchmedia to bypass error.
+Object.defineProperty(window, 'matchMedia', {
+  writable: true,
+  value: jest.fn().mockImplementation(query => ({
+    matches: false,
+    media: query,
+    onchange: null,
+    addListener: jest.fn(), // deprecated
+    removeListener: jest.fn(), // deprecated
+    addEventListener: jest.fn(),
+    removeEventListener: jest.fn(),
+    dispatchEvent: jest.fn(),
+  })),
+});
+
+describe('<FamilyIntake/> test suite ', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  test('Sanity Check: renders without error', () => {
+    render(<FamilyIntake />);
+  });
+
+  test('All inputs reflect the value the user has typed', () => {
+    render(<FamilyIntake />);
```

Comment:
I wonder if this breaks due to the fact that you're using React Router. It is required to wrap Router component in your testing suite when running test IF that component relies on the router to mounted to the DOM.

Reply:
Could you elaborate on this a bit more?